README.md

```diff
@@ -1,15 +1,133 @@
 
 Developers don’t have to worry about the underlying infrastructure, whether they’re developing their app locally, deploying it to a [Managed Runtime](https://developer.salesforce.com/docs/commerce/pwa-kit-managed-runtime/guide/mrt-overview.html) environment, or testing the app live.
 
 ## Requirements
 
+Before you begin, ensure that you have the following accounts set up:
+
+-   [SFCC B2C Commerce account](https://dashboard.elasticpath.com/login)
+-   [Builder.io Account](https://builder.io)
 -   Node 14
 -   npm 6.14.4 or later
 
-## Getting Started
+
+### Getting started with Builder.io :
+      - [1: Create an account for Builder.io](#1-create-an-account-for-builderio)
+      - [2: Your Builder.io private key](#2-your-builderio-private-key)
+      - [3: Clone this repository and initialize a Builder.io space](#3-clone-this-repository-and-initialize-a-builderio-space)
+
+#### 1: Create an account for Builder.io
+
+Before we start, head over to Builder.io and [create an account](https://builder.io/signup).
+
+#### 2: Your Builder.io private key
+
+Head over to your [organization settings page](https://builder.io/account/organization?root=true) and create a
+private key, copy the key for the next step.
+
+- Visit the [organization settings page](https://builder.io/account/organization?root=true), or select
+  an organization from the list 
+
+![organizations drop down list](https://cdn.builder.io/api/v1/image/assets%2Fd1ed12c3338144da8dd6b63b35d14c30%2F573b9e0ce1fc43c09a069fd4976be47c)
+
+- Click "Account" from the left hand sidebar
+- Click the edit icon for the "Private keys" row
+- Copy the value of the auto-generated key, or create a new one with a name that's meaningful to you
+
+
+![Example of how to get your private key](https://cdn.builder.io/api/v1/image/assets%2Fd1ed12c3338144da8dd6b63b35d14c30%2Fd824c5a4ddc3436e806d4b7305961d96)
+
+![Example of how to get your private key](https://cdn.builder.io/api/v1/image/assets%2Fd1ed12c3338144da8dd6b63b35d14c30%2F7a121a72a0f749a1a9d154b96914be26)
+
+
+## Initialize a space with this starter kit
+Follow the steps below to initialize a blank Builder.io space with the starter kit features and config.
+#### *Note: If you want to clone/import our pre created templates for models and custom targeting, see the Cloning/Importing section below and then come back to this section.*
+
+#### Clone this repository and initialize a Builder.io space
+
+Next, we'll create a copy of the starter project, and create a new
+[space](https://www.builder.io/c/docs/spaces) for it's content to live
+in.
+
+In the example below, replace `<private-key>` with the key you copied
+in the previous step, and change `<space-name>` to something that's
+meaningful to you -- don't worry, you can change it later!
+
+Note:
+if you're only interested in using this starter for a landing page use this command instead:
+
+```
+builder create --key "<private-key>" --name "<space-name>" --input builder-landing-page-only --debug
```

Comment:
this is not accurate, please clone space and add it to the file and update this part